dev_apps/music-nga/js/endpoint.js

```diff
@@ -184,7 +186,9 @@ function setShuffleSetting(shuffle) {
 
 function getAlbums() {
   return new Promise((resolve) => {
-    Database.enumerateAll('metadata.album', null, 'nextunique', albums => resolve(albums));
+    Database.enumerable.then(() => {
+      Database.enumerateAll('metadata.album', null, 'nextunique', albums => resolve(albums));
+    });
```

Comment:
I'm not a big fan of this. I think we should perform the `enumerable` check inside `enumerateAll` so that users of the API don't have to check themselves.